blackbelt/handle_github.py

```diff
@@ -270,6 +270,30 @@ def deploy(pr_url):
     try:
         ticket_id = get_pr_ticket_id(merge_info['description'])
         move_to_deployed(card_id=ticket_id, comment=comment)
+        create_release(ref=merge_info['sha'], payload='', description="Deployed to production")
     except ValueError:
         if click.prompt("Moving card failed. Open PR in browser?", default=True):
             webbrowser.open(merge_info['html_url'])
+
+def create_release(ref, payload, description):
+    """ Create release in github after deploy to production """
+
+    url = "https://api.github.com/repos/%(owner)s/%(name)s/deployments"
+
+    payload = {
+        'ref': ref,
+        'payload': payload,
+        'description': description
+    }
```

Comment:
@abtris I can see the redundancy in variable name i.e. payload. You might want rename one or another to something else.